samples/bucketLock.js

```diff
@@ -34,7 +34,9 @@ async function setRetentionPolicy(bucketName, retentionPeriod) {
     .bucket(bucketName)
     .setRetentionPeriod(retentionPeriod);
   console.log(
-    `Bucket ${bucketName} retention period set for ${metadata.retentionPolicy.retentionPeriod} seconds.`
+    `Bucket ${bucketName} retention period set for ${
```

Comment:
Going to want to `rm -rf node_modules package-lock.json` && `npm run fix` here.  Prettier made an update in a patch release I think that's causing some grief.

Reply:
Whoops, well this is embarrassing 😱